app/migrate.go

```diff
@@ -60,150 +38,151 @@ func GenesisDocFromFile(genDocFile string) (*tmtypes.GenesisDoc, error) {
 
 // MigrateGenesisCmd returns a command to execute genesis state migration.
 // nolint: funlen
+// TODO: implement for support cosmos-sdk (v0.46.10)
 func MigrateGenesisCmd() *cobra.Command {
```

Comment:
@taobun Hi, want to seek confirmation about this migrate command. Could we remove it?

Reply:
Just remove it (legacy migrate method)

Reply:
thanks.